src/items/enumerations.md

```diff
@@ -56,22 +56,58 @@ a = Animal::Cat { name: "Spotty".to_string(), weight: 2.7 };
 ```
 
 In this example, `Cat` is a _struct-like enum variant_, whereas `Dog` is simply
-called an enum variant. Each enum instance has a _discriminant_ which is an
-integer associated to it that is used to determine which variant it holds. An
-opaque reference to this discriminant can be obtained with the
-[`mem::discriminant`] function.
+called an enum variant.
 
-## Custom Discriminant Values for Fieldless Enumerations
+An enum where no constructors contain fields are called a
+*<a id="field-less-enum">field-less enum</a>*.
```

Comment:
I think the field-less term got a little confusing, and it's still not clear to me, does it also cover empty constructors, like:

```rust
enum Fieldless {
    Tuple(),
    Struct{},
    Unit,
}
```

If so, perhaps that needs to be made explicit?

Reply:
Yeah, I will add that as an example.